### ISSUE TITLE
baked styles refactor - examples

### ISSUE THREAD

**Issue by lwtchu** [openstax/cnx-recipes#4677]
- [x] Create general components for title parts (icon, label etc.). They are styled the same for solution and commentary.
- [x]  Corn - Example problem and solution - Should we create separate shape for exercises in examples in exercises files? If not, where exercise components should be defined - in examples or exercises files? - LC: I think this (separate shape for exercises in examples) might be best? Happy to discuss more in depth once we get to this
- [x] Create files for Examples in Carnival. Now they are styled using `IconNoteShape`.

### What changes are done:
- New shape `ExerciseFromExampleShape` that styles problem, solution and commentary title in examples. Applied to Corn.
- New files for examples in Carnival.
- Notes components from Carnival that were used only for examples are removed from `IconNoteShape` and `LogoNoteIconShape` (in this one they were probably just copied from IconNoteShape). Maybe it will be good to test these types of notes as well during Example testing.

**Comment by wilkus27** [comment]
@lwtchu In Cardboard there is additional component in Examples `NoteTitle`. I checked and it appears in Astronomy:

![image](https://user-images.githubusercontent.com/40228854/170274034-b40c6305-1b2f-4c01-b043-e620e6c7f0f4.png)

Is it ok, that part of the example is a note? Other parts are just paragraphs with h4 title. Does it gonna be changed?

**Comment by lwtchu** [comment]
@wilkus27 yeah i think it's okay for now 🤔  I think ... are there are other books that have this also like chemistry, college physics, university physics, statistics, intro stats 🤔  

if in the future, we remove the note and set it as a para title, would there be almost no change?

**Comment by wilkus27** [comment]
@lwtchu I just found out that in Carnival examples are styled using note shape. I will create separate files for them to be consistent with Cardboard and Corn. And as I checked, e.g. in Chemistry **Answer** is created as a note like in Astronomy, so this is a common mark up, not one instance as I thought.

Also I want to make sure, are there any examples in Cosmos? I don't see any inputs in files in Styles or Cookbook, but I might missing something :thinking:
